app/assets/javascripts/admin/ck_editor_initiate.js

```diff
@@ -0,0 +1,5 @@
+ClassicEditor
+  .create( document.querySelector( '#editor' ) )
```

Comment:
also add 2-3 custom options here, so that you understand how to build custom WYSIWYG